cpp/src/arrow/engine/substrait/extension_set.cc

```diff
@@ -355,6 +355,68 @@ ExtensionIdRegistry::SubstraitCallToArrow kSimpleSubstraitToArrow =
   return ::arrow::compute::call(std::string(call.id().name), std::move(args));
 };
 
+ExtensionIdRegistry::SubstraitAggregateToArrow DecodeBasicAggregate(
```

Comment:
TODO: Revert this change after https://github.com/apache/arrow/pull/35608 is merged

Reply:
Done